sys/riotboot/Makefile.include

```diff
@@ -1,7 +1,15 @@
-# Indicate the reserved space for a header, 256B by default
-# Notice that it must be 256B aligned. This is restricted by
-# the Cortex-M0+/3/4/7 architecture
-RIOTBOOT_HDR_LEN ?= 0x100
+# Indicate the reserved space for a header.
+# Notice that it must be at least 256B aligned. This is restricted by
```

Comment:
Could you update to remove the previous `Notice that it must be at least 256 aligned` and include this part in the following part. Currently it has two different informations mixed.